QUICK_SETUP.md

```
@@ -0,0 +1,275 @@
+```
+
+### **2. Create Backend Environment**
+```bash
+cd backend
+cp .env.example .env
+```
+
+Edit `backend/.env` and fill in these sections:
+
+**A. Local Models** (Copy-paste these - they're ready to use):
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)